src/components/ApplicationLayout/ApplicationLayout.tsx

```diff
@@ -155,71 +155,74 @@ const ApplicationLayout = <
           }}
         />
       </AppNavigationBar>
```

Comment:
The `AppNavigationBar` should also be hidden if there is no menu to display otherwise the top bar appears on mobile:

<img width="389" alt="Screenshot 2024-07-18 at 10 05 12 AM" src="https://github.com/user-attachments/assets/d7ef6de3-94dd-406b-a325-e316b3a31251">